web/static/app.css

```
@@ -646,6 +646,36 @@ a.name .title { flex: 1 1 auto; min-width: 0; }
 }
 .project-description.markdown > :first-child { margin-top: 0; }
 .project-description.markdown > :last-child { margin-bottom: 0; }
+.projects-heading { display: flex; align-items: center; justify-content: space-between; gap: 1rem; }
+.projects-heading h1 { margin-bottom: 0; }
+.project-create-panel { max-width: 46rem; margin: 1rem 0; }
+.project-key-preview { margin: -0.45rem 0 0.8rem; font-size: 0.86rem; }
+.project-state-tabs { display: flex; gap: 1.25rem; margin: 1.25rem 0 0.8rem; border-bottom: 1px solid var(--line); }
+.project-state-tabs a { padding: 0.55rem 0.1rem; color: var(--muted); text-decoration: none; }
+.project-state-tabs a.active { color: var(--accent); border-bottom: 2px solid currentColor; font-weight: 650; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)